cmd/kubeadm/app/cmd/join.go

```diff
@@ -426,7 +426,7 @@ func (j *joinData) TLSBootstrapCfg() (*clientcmdapi.Config, error) {
 	if j.tlsBootstrapCfg != nil {
 		return j.tlsBootstrapCfg, nil
 	}
-	klog.V(1).Infoln("[preflight] Discovering cluster-info")
+	klog.V(1).Infoln("[preflight] discovering cluster-info")
```

Comment:
we decided that all messages that are in the form of `[area] Doing something` to have the "Doing" part starting with upper case.

Reply:
Thanks @neolit123 ! 
Do we need to capitalize the first letter for all messages?
Or just leave them as they are?

Reply:
possibly, yes.
at least those that start with `[area]` should be.